### PR TITLE
docs: add info about new option "autoWebviewName"

### PR DESCRIPTION
**Pull request by DudaGod**
Add info about new option `autoWebviewName` which will be added here - https://github.com/appium/appium-android-driver/pull/770

### CODE REVIEW

**Comment by mykola-mokhnach** [comment]
We should publish a new version of appium-android-driver package and bump it in the current package.json
I'll let you know when it is published

File: README.md [hunk 1]
```diff
@@ -152,7 +152,9 @@ appium:mjpegScreenshotUrl | The URL of a service that provides realtime device s
 
 Capability Name | Description
 --- | ---
-appium:autoWebview | If set to `true` then UiAutomator2 driver will try to switch to the first available web view after the session is started. `false` by default.
+appium:autoWebview | If set to `true` then UiAutomator2 driver will try to switch to the web view with name `WEBVIEW_ + appium:appPackage` after the session is started. For example, if `appium:appPackage` capability is set to `com.mypackage` then `WEBVIEW_com.mypackage` will be used. `false` by default.
+appium:autoWebviewName | Set the name of webview context in which UiAutomator2 driver will try to switch if `autoWebview` capability is set to `true`. Has priority over using the `appium:appPackage` value in webview name. For example, if `appium:autoWebviewName` capability is set to `myWebviewName` then `WEBVIEW_myWebviewName` will be used. Unset by default.
```
Comment (mykola-mokhnach) [inline README.md:156]: It would also be useful to mention in which driver version this cap has been introduced for the first time.

**Comment by mykola-mokhnach** [comment]
Thanks @DudaGod 
Could you please resolve the conflict before I merge?

**Comment by DudaGod** [comment]
> Could you please resolve the conflict before I merge?

Done.

**Review by KazuCocoa — APPROVED** [review]

**Comment by DudaGod** [comment]
When will the appium released with these changes? It looks like that appium@1 is no longer released - https://github.com/appium/appium/releases/tag/v1.22.1. So how I can use these changes?

**Comment by DudaGod** [comment]
@mykola-mokhnach @KazuCocoa guys, can you tell me when I can use these changes in appium? I asked about this a couple of days ago:

> When will the appium released with these changes? It looks like that appium@1 is no longer released - https://github.com/appium/appium/releases/tag/v1.22.1. So how I can use these changes?

**Comment by DudaGod** [comment]
> It is always available via Appium 2 once a new driver version is released, but not for Appium v1

But I don't see new version of appium@2. Latest available version is 2.0.0-beta.46:
```
❯❯❯ npm view appium
// ...
dist-tags:
beta: 1.23.0-beta.0  fake: 1.20.0-fake.0  latest: 1.22.3       next: 2.0.0-beta.46  rc: 1.22.3-rc.1
``` 

Which published 2 weeks ago - https://github.com/appium/appium/releases/tag/appium%402.0.0-beta.46.
Moreover when will the stable version come out?

**Comment by DudaGod** [comment]
> Appium 2 needs to install the necessary drivers as https://github.com/appium/appium#drivers You can install/update individual drivers independent from the appium server itself in v2.

I'm not very comfortable using the beta version, but according to the [CHANGELOG](https://github.com/appium/appium/blob/master/packages/appium/CHANGELOG.md) it looks like there are no global changes. Maybe there is already some kind of roadmap with information about what else needs to be changed before releasing a stable version?

**Comment by DudaGod** [comment]
> but according to the [CHANGELOG](https://github.com/appium/appium/blob/master/packages/appium/CHANGELOG.md) it looks like there are no global changes

I found more detailed description of changes here - https://appium.github.io/appium/docs/en/2.0/guides/migrating-1-to-2/

**Comment by KazuCocoa** [comment]
https://github.com/appium/appium/blob/master/packages/appium/docs/en/guides/migrating-1-to-2.md is the page's base.

> what else needs to be changed before releasing a stable version?

Afaik, the main things are documentation work

**Comment by DudaGod** [comment]
> Afaik, the main things are documentation work

Okay, thank you.

**Pull request merged**